src/main.rs

```diff
@@ -84,7 +84,13 @@ fn main() {
                 eprintln!();
                 eprintln!("Try running without the --native-tls flag.");
             }
-            process::exit(1);
+            if msg == "operation timed out" {
```

Comment:
There's a more robust way to detect this:
```rust
            if let Some(err) = err.downcast_ref::<reqwest::Error>() {
                if err.is_timeout() {
                    [...]
                }
            }
```

We could also do that for the "Too many redirects" case by turning it into a custom error type, but there we control the error message ourselves so it's not necessary.